src/pages/studyView/tabs/CNSegments.tsx

```diff
@@ -28,12 +28,18 @@ export default class CNSegments extends React.Component<
 > {
     @observable renderingComplete = false;
     @observable segmentTrackMaxHeight: number | undefined;
+    private lastSelectedLocus: string | undefined = undefined;
 
     constructor(props: { store: StudyViewPageStore }) {
         super(props);
         this.segmentTrackMaxHeight = WindowStore.size.height * 0.7;
     }
 
+    @autobind
```

Comment:
@action.  i know it's academic here but want to make things consistent so new devs will adopt habit.

Reply:
@alisman its not observable though